packages/config/src/layer2s/stealthchain.ts

```diff
@@ -0,0 +1,39 @@
+import { ProjectId } from '@l2beat/shared-pure'
+
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from './common'
+import { Layer2 } from './types'
+
+export const stealthchain: Layer2 = {
+  isUpcoming: true,
+  type: 'layer2',
+  id: ProjectId('stealthchain'),
+  display: {
+    name: 'StealthChain',
+    slug: 'stealthchain',
+    description:
+      'Degen Stealth Launchpad, Cross Chain Token Bridge & LP Pool',
+    purpose: 'Degen,Launchpad',
+    category: 'OP Stack',
+    links: {
+      websites: ['https://steathchain.org'],
+      apps: ['https://bridge.test.steathchain.org'],
+      documentation: ['https://docs.steathchain.org'],
```

Comment:
These links are dead

- `steathchain.org` is up available to be bought
- `stealthchain.org` was bought today

So I presume it's a typo, but make sure that the website and docs work after DNS propagates

Reply:
Yes Typo Links Updated.

Thank you